### ISSUE TITLE
bug: docker run -it --entrypoint /bin/bash apeworx/ape:0.6.10 project directory is not root 

### ISSUE THREAD

**Issue by sabotagebeats** [ApeWorX/ape#1476]
### Environment information

- OS: ubuntu wsl2 windows 11
- Python Version: docker
- `ape` and plugin versions: docker

### What went wrong?

Please include information like:

- what command you ran
```
~/ape/ape$ docker build -t ape . --no-cache
[+] Building 158.4s (17/17) FINISHED                                                                             
 => [internal] load .dockerignore                                                                           0.0s
 => => transferring context: 65B                                                                            0.0s
 => [internal] load build definition from Dockerfile                                                        0.0s
 => => transferring dockerfile: 1.79kB                                                                      0.0s
 => [internal] load metadata for docker.io/library/python:3.9-slim                                          0.9s
 => [internal] load metadata for docker.io/library/python:3.9                                               0.9s
 => [internal] load build context                                                                           0.0s
 => => transferring context: 44B                                                                            0.0s
 => CACHED [stage-1 1/7] FROM docker.io/library/python:3.9-slim@sha256:5f0192a4f58a6ce99f732fe05e3b3d00f12  0.0s
 => [builder 1/4] FROM docker.io/library/python:3.9@sha256:6e9c45244cf9ef21fe9310f6c7bab754c5e4d23c819421f  0.0s
 => CACHED [builder 2/4] WORKDIR /wheels                                                                    0.0s
 => [stage-1 2/7] RUN useradd --create-home --shell /bin/bash harambe                                       0.8s
 => [builder 3/4] COPY ./recommended-plugins.txt ./recommended-plugins.txt                                  0.0s
 => [builder 4/4] RUN pip install --upgrade pip     && pip install wheel     && pip wheel -r ./recommende  67.5s
 => [stage-1 3/7] COPY --from=builder /wheels /wheels                                                       0.2s 
 => [stage-1 4/7] COPY ./recommended-plugins.txt ./recommended-plugins.txt                                  0.0s 
 => [stage-1 5/7] RUN pip install --upgrade pip     pip install --no-cache-dir --find-links=/wheels -r ./  84.5s 
 => [stage-1 6/7] WORKDIR /home/harambe/project                                                             0.0s 
 => [stage-1 7/7] RUN chown -hR harambe:harambe /home/harambe                                               0.7s 
 => exporting to image                                                                                      3.7s
 => => exporting layers                                                                                     3.7s
 => => writing image sha256:fead3634bdd1028838a99a6d54d01a1477550412e572995434dd285736d65a65                0.0s
 => => naming to docker.io/library/ape                                                                      0.0s
~/ape/ape$ docker run -it --entrypoint /bin/bash ape
harambe@ef740bd7a6e8:~/project$ ls -la
total 12
drwxr-xr-x 1 harambe harambe 4096 Jun 12 18:37 .
drwxr-xr-x 1 harambe harambe 4096 Jun 12 18:37 ..
harambe@ef740bd7a6e8:~/project$ exit
$ docker run -it --entrypoint /bin/bash apeworx/ape:0.6.10
harambe@62e33539fe9e:~/project$ ls -la
total 8
drwxr-xr-x 2 root    root    4096 May 25 21:51 .
drwxr-xr-x 1 harambe harambe 4096 May 25 21:51 ..
```

```
$ docker run -it ape --version
0.6.10
$ docker run -it apeworx/ape:0.6.10 --version
0.6.10
```

### How can it be fixed?
ensure github action for building apeworx/ape:latest and apeworx/ape:version is using --no-cache

**Comment by sabotagebeats** [comment]
I believe this is causing silverback not to install in docker properly when I use the apeworx/ape:latest image as a base

**Comment by sabotagebeats** [comment]
related: https://github.com/SilverBackLtd/silverback/pull/8

**Comment by sabotagebeats** [comment]
thanks to @antazoey sleuthing, turns out https://github.com/ApeWorX/ape/pull/1448 is not a part of 0.6.10 which is causing this bug. Once that PR is merged into the next release, 0.6.11 will not have this bug.
